Dockerfile

```diff
@@ -6,6 +6,13 @@ WORKDIR "${INSTALL_DIR}"
 COPY . "${INSTALL_DIR}"
 
 RUN apk add --update make gcc g++ git openssl
+
+# Update the package index and upgrade existing packages
+RUN apk update && apk upgrade
```

Comment:
can you please club all the RUN commands together? That will reduce the size

Reply:
sure